### ISSUE TITLE
batchnorm problem

### ISSUE THREAD

**Issue by qgh1223** [Y-P-Zhang/LRF-Net-implemented-in-mmdetection#1]
I used general inference code in mmdetection.But I met Expected more than 1 value per channel when training, got input size torch.Size([1, 256, 1, 1]) problem.How to solve?Can you give the visualize code

**Comment by qgh1223** [comment]
I get the bug.I only need to add model.eval()
